testutil/beaconmock/options.go

```diff
@@ -489,6 +491,10 @@ func defaultMock(httpMock HTTPMock, httpServer *http.Server, clock clockwork.Clo
 		BlockAttestationsFunc: func(ctx context.Context, stateID string) ([]*eth2p0.Attestation, error) {
 			return []*eth2p0.Attestation{}, nil
 		},
+		NodePeerCountFunc: func(ctx context.Context) (eth2wrap.PeerCount, error) {
+			//nolint:gosec
+			return eth2wrap.PeerCount{Connected: rand.Int() % 100}, nil
```

Comment:
i'm open to this being constant

Reply:
yes please, zero `eth2wrap.PeerCount` struct.